src/java/com/zimbra/oauth/handlers/impl/OAuth2Handler.java

```diff
@@ -60,6 +60,21 @@
  * @copyright Copyright © 2018
  */
 public abstract class OAuth2Handler {
+    // for relay encoding
+    public enum RelayEnum {
+        RELAY("relay"),
```

Comment:
These constants exist in the project.
https://github.com/Zimbra/zm-oauth-social/blob/develop/src/java/com/zimbra/oauth/utilities/OAuth2HttpConstants.java#L38

Also "relay" isn't used anymore since the tickets with the ds type updates - it's now "state" on the authorize call, and expecting whatever the social service uses (typically state - but configurable by each one).

Reply:
this enum is not used anywhere, so removed. I added it while i was testing JweUtil to pass the relay values but i removed that part of code.